### ISSUE TITLE
Write up 'Chem Ed' Paper on Project

### ISSUE THREAD

**Issue by alintheopen** [OpenSourceMalaria/OSM_To_Do_List#308]
We'd like to write-up the work OSM and SGS have done and outline how others can be involved with OSM or open projects. The paper will be written collaboratively on an open platform such as GitHub or GDocs.

**Comment by alintheopen** [comment]
Open Access Journals for Science/Chemical Education:

Chemistry Education Research and Practice (RSC)
http://pubs.rsc.org/en/journals/journalissues/rp#!recentarticles&adv

PLoS One has a (small) Science Education Section
http://www.plosone.org/browse/science_education

World Journal of Chemical Education
http://www.sciepub.com/journal/WJCE

**Comment by mattodd** [comment]
Open access option at J Chem Ed?

**Comment by alintheopen** [comment]
I've asked (https://twitter.com/all_isee/status/613937855673450496)

Also, I'm loathed to mention IF, but I am going there.....
J Chem Ed = 1.106
Chem. Educ. Res. Pract. = 1.309

**Comment by mattodd** [comment]
i.e. they're both about 1.

On 25 June 2015 at 15:17, alintheopen <notifications@github.com> wrote:

>  I've asked (https://twitter.com/all_isee/status/613937855673450496)
>
> Also, I'm loathed to mention IF, but I am going there.....
> J Chem Ed = 1.106
> Chem. Educ. Res. Pract. = 1.309
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/308#issuecomment-115108132>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/chemistry/research/todd.html | W
http://opensourcemalaria.org/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by ErinSheridan** [comment]
If J Chem Ed doesn't have an open access option (and I haven't seen any sign that they do), Chemistry Education Research & Practice is a good option.

Also I found a couple of related papers that make reasonable background reading.
[Pueyo 2013.pdf](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/283/Pueyo.2013.pdf)
[adams 2008.pdf](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/282/adams.2008.pdf) 
[Carriere 1997.pdf](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/284/Carriere.1997.pdf)

**Comment by alintheopen** [comment]
Yes. But one is completely open access and the other is author choice (
https://twitter.com/acspublications/status/614054368279703553)

**Comment by PatrickThomson** [comment]
Hi all,

very very briefly popping out of retirement as I have, in the last year, fully committed myself to a career in chemical education and pedagogical research and am still training up on that front; I've been busy embedding in the field, including conferences and journals, so I might be able to lend some perspective.


The paper may not be accepted to the RSC's flagship educational journal CERP - this journal is a victim of its own success and in practice only usually publishes "Research" into chemistry education - the "Practice" has fallen by the wayside and picked up by others and it's no coincidence that all three of the relevant references given by @ErinSheridan were in J Chem Ed. Unless the paper has quantitative education-specific research, it's likely to be referred on somewhere else. However, CERP is open access and the most prestigious chemistry education journal so if you can get in it's worth a go. If needed, I can make some informal inquiries as to feasibility (I know some of the advisory board and am literate in both education research and OSM). 

Re: impact, last summer CERP climbed to an impact factor of 2, which elicited unironic whoops from the Tertiary Education Group AGM - it takes some getting adjusting but IF is a field-specific metric as you know, and 2 is stunning in chemical education.  

If you want an uncomplicated paper to get our project to the attention of the chemical education field, J Chem Ed is the de-facto publishing ground for chemical education practice. It's something of a shame that a lot of UK and European research ends up under the umbrella of the American Chemical Society, with all the associated trappings of open access policy that brings with it - the usual ACS open access arrangements hold with J Chem Ed; they're amenable to OA as it's often a funding requirement, but I can't imagine it's cheap. 

Both journals are required reading for chemistry pedagogues at university level, and the audience will be extremely similar. If you want to reach schoolteachers, the magazine "Education in Chemistry" is the equivalent of Chemistry World, for secondary & HE teachers, and a writeup there will be the best thing you can do to increase OSM's uptake in schools.



In summary: Patrick's still in hibernation, but starting to do educational research and willing to check to see if there's any chance of getting your paper into CERP.
